specifications/api/swagger.yaml

```diff
@@ -2097,18 +2097,50 @@ components:
         fingerprint: *assetFingerprint
         metadata: *assetMetadata
 
+    ApiWalletMigrationBalance: &ApiWalletMigrationBalance
+      type: object
+      required:
+        - ada
+        - assets
+      properties:
+        ada: *amount
+        assets: *walletAssets
+
     ApiWalletMigrationInfo: &ApiWalletMigrationInfo
       type: object
       required:
-        - migration_cost
-        - leftovers
+        - selections
+        - total_fee
+        - balance_leftover
+        - balance_selected
       properties:
-        migration_cost:
-          <<: *amount
-          description: Total amount which will be paid as fees for the migration.
-        leftovers:
+        selections:
```

Comment:
This field could have a description too.

Reply:
> This field could have a description too.

That's a good point!

I've added a description in this updated commit: 202e0240bfbfcdb88bbdb3ef780dce36a13cb890.